src/engine/duel/animations/screen_effects.asm

```diff
@@ -252,11 +252,11 @@ DistortScreen:
 
 ; each value is applied for 8 "ticks" of wScreenAnimDuration
 ; starting from the last and running backwards
-.BGScrollModData
+.BGScrollModData:
 	db 4, 3, 2, 1, 1, 1, 1, 2
 
 Func_1ce03:
-	cp DUEL_ANIM_158
+	cp DUEL_ANIM_158_UNUSED
 	jr z, .asm_1ce17
 	sub $96
```

Comment:
I think this `$96` would make for a good constant, just like `DUEL_SCREEN_ANIMS` but to mark the start of `DUEL_ANIM_SET_SCREEN`.

There's another relevant `$96` on line 485 of core.asm.

Reply:
Oh, this appears to be in your tcg2 PR as `DUEL_SPECIAL_ANIMS`. I can just be patient :)